prusti-common/src/vir/optimizations/functions/simplifier.rs

```diff
@@ -46,134 +50,118 @@ impl ExprSimplifier {
                 argument:
                     box ast::Expr::Const(ast::ConstExpr {
                         value: ast::Const::Bool(b),
-                        ..
+                        position: inner_pos,
                     }),
                 position: pos,
             }) => ast::Expr::Const(ast::ConstExpr {
                 value: ast::Const::Bool(!b),
                 position: pos,
-            }),
+            })
+            .set_default_pos(inner_pos),
```

Comment:
What does this do? If it's a constant Boolean, there shouldn't be any more position to override with `set_default_pos`, right?